lighthouse-core/gather/gatherers/seo/tap-targets.js

```diff
@@ -115,7 +121,7 @@ function filterClientRectsWithinAncestorsVisibleScrollArea(element, clientRects)
     const parentBCR = parent.getBoundingClientRect();
     clientRects = clientRects.filter(cr => rectContains(parentBCR, cr));
   }
-  if (parent.parentElement && parent.parentElement.tagName !== 'BODY') {
+  if (parent.parentElement && parent.parentElement !== document.documentElement) {
```

Comment:
just curious, what specific dom structure did this fix :)

Reply:
Can't find it anymore, but that code's gone now anyway.